src/lender/tranche/senior_tranche.sol

```diff
@@ -52,12 +52,10 @@ contract SeniorTranche is Tranche, Interest {
 
     function file(bytes32 what, uint ratePerSecond_) public note auth {
          if (what ==  "rate") {
-            drip();
```

Comment:
the most important change in this PR

Reply:
I think we still need to drip in case file is used to update the rate